app/screens/PartyDetailsScreen/PartyDetailsScreen.tsx

```diff
@@ -0,0 +1,176 @@
+import { useSchedule } from "../../services/api"
+import { formatDate } from "../../utils/formatDate"
+import { openMap } from "../../utils/openMap"
+
+const partyBlob = require("../../../assets/images/party-shape.png")
+const partyCurve = require("../../../assets/images/party-curve.png")
+const squarespaceLogo = require("../../../assets/images/squarespace-logo.png")
+
+const SCREEN_WIDTH = Dimensions.get("screen").width
+
+export const PartyDetailsScreen: FC<StackScreenProps<AppStackParamList, "PartyDetails">> = ({
+  route: { params },
+}) => {
+  const { scheduleId } = params
+  const { data: scheduleData } = useSchedule()
+  const schedule = scheduleData?.find((s) => s._id === scheduleId)
+
+  const scrollY = useSharedValue(0)
+  const [headingHeight, setHeadingHeight] = React.useState(0)
+
+  const scrollHandler = useAnimatedScrollHandler({
+    onScroll: (event) => {
+      scrollY.value = event.contentOffset.y
+    },
+  })
+
+  const { bottom: paddingBottom } = useSafeAreaInsets()
+
+  const openLink = (destination: string) =>
+    destination.startsWith("https") ? openLinkInBrowser(destination) : openMap(destination)
+
+  if (schedule === undefined) return null
+
+  const title = schedule.name
+  const subtitle = formatDate(schedule["day-time"], "MMM d, h:mm a")
+
+  return (
+    <Screen safeAreaEdges={["top", "bottom"]} style={$root}>
+      <TalkDetailsHeader {...{ title, subtitle, scrollY, headingHeight }} />
+
+      <Animated.ScrollView
+        style={[$scrollView, { paddingBottom }]}
+        scrollEventThrottle={16}
+        onScroll={scrollHandler}
+        showsVerticalScrollIndicator={false}
+      >
+        <View style={$container}>
+          <View style={$headingContainer}>
+            <Text
+              preset="screenHeading"
+              style={$title}
+              text={title}
+              onLayout={({
+                nativeEvent: {
+                  layout: { height },
+                },
+              }) => {
+                setHeadingHeight(height)
+              }}
+            />
+            <Text preset="companionHeading" style={$subtitle} text={subtitle} />
+          </View>
+          <View style={$containerSpacing}>
+            <Image source={partyCurve} style={$partyCurve} />
+            <Image source={{ uri: "https://picsum.photos/315" }} style={$partyImage} />
+            <Image source={partyBlob} style={$partyBlob} />
+          </View>
+
+          <View style={$detailsContainer}>
+            <Text style={$bodyText} text={schedule["break-party-description"]} />
+          </View>
+
+          <View style={$containerSpacing}>
+            <Text preset="eventTitle" style={$aboutHeading} text="Hosted by" />
+            <Image source={squarespaceLogo} />
+          </View>
+
+          <View style={$containerSpacing}>
+            <Text preset="eventTitle" style={$aboutHeading} text="Location" />
+            <Text style={[$bodyText, $boldText]} text="The Armory" />
```

Comment:
There is some heading preset to use for this so it looks more like the mock up. The font-weight applied here isn't actually bolding it.

![image](https://user-images.githubusercontent.com/374022/211365482-b91afe0e-00fd-4afc-80ee-7879728186a7.png)